ci-operator/step-registry/redhat-developer/rhdh/osd-gcp/claim-cluster/OWNERS

```diff
@@ -0,0 +1,18 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# Fetched from https://github.com/redhat-developer/rhdh root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- josephca
+- rnapoles-rh
```

Comment:
We can safely remove `rnapoles-rh` from all of our CI jobs. I removed him from most of them in https://github.com/openshift/release/commit/15a0980927e0b631756f7fa88fce1c5a1baf2261